e2e-tests/specs/relay-general-functionality.spec.ts

```diff
@@ -1,9 +1,9 @@
 import test, { expect }  from '../fixtures/basePages'
-import { checkAuthState, defaultScreenshotOpts, ENV_DOMAINS } from '../e2eTestUtils/helpers';
+import { checkAuthState, defaultScreenshotOpts } from '../e2eTestUtils/helpers';
 
 // using logged in state outside of describe block will cover state for all tests in file
 test.use({ storageState: 'state.json' })
-test.describe('Free - General Functionalities, Desktop', () => {
+test.describe.fixme('Free - General Functionalities, Desktop', () => {
```

Comment:
Same here, correct me if I'm wrong but I believe fixme won't run the test until its fixed?

Reply:
correct, this is out of scope for this PR